components/google_sign_in/common/DEPS

```diff
@@ -0,0 +1,7 @@
+include_rules = [
+  "+content/public/browser",
```

Comment:
`common` cannot include `content`.

pls move everything to `browser` folder. Use `common` only for parts that may be shared between browser/renderer (for. ex, features).